docker-compose.yml

```diff
@@ -10,13 +10,14 @@ services:
     restart: unless-stopped
     environment:
       - ROOT_PATH=/backend
-      - VOTING_SERVICE_PATH=host.docker.internal:8080/voting-service-api
-      - STATE_VECTOR_PATH=host.docker.internal:8080/statevector
-      - VOTING_PROCESS_MANAGER_HOST=host.docker.internal:8080
-      - VOTING_PROCESS_MANAGER_HOST_SOCKET_PATH=voting-process-manager-api/ws/socket.io
-      - VOTING_PROCESS_MANAGER_PATH=host.docker.internal:8080/voting-process-manager-api
+      - HYPERTEXT_PROTOCOL=https://
```

Comment:
Tieto všetky env neviem, či chceme meniť na team17...

Reply:
podla mna to je dost pekna ficurka lebo mna nikdy ked som robil daco na VT nezaujmalo ako G bezi ci to je u mna lokalne alebo na serveri, tak som tam nehal zakomentovane tie veci z team17 ale to co si chcel som nehal odkomentovane